diesel_cli/src/cli.rs

```diff
@@ -289,6 +302,12 @@ pub fn build_cli() -> Command {
                 .action(clap::ArgAction::Append)
                 .number_of_values(1)
                 .help("A list of derives to implement for every automatically generated SqlType in the schema, separated by commas."),
+        )
+        .arg(
+            Arg::new("print-schema-key")
```

Comment:
I wonder if just `schema-key` might be the better name here?

Reply:
Thanks, that would be better.